### ISSUE TITLE
If configure marker enabled to false and 'allowPointSelect' to true, click on a point to select it, sometimes the point disappear.

### ISSUE THREAD

**Issue by zhangmengjie-amy** [highcharts/highcharts#19817]
In a line chart:
step1: Configure 'allowPointSelect' to true.,
step2: Configure marker enabled to false,

#### Expected behaviour
Hover on a point and click to select it, firstly the point is selected, and after clicking the second point, the point is selected.

#### Actual behaviour
Hover on a point and click to select it, firstly the point is selected, and after clicking the second point, the point disappears.

#### Live demo with steps to reproduce

https://jshare.com.cn/edit/public/pmnweyln

#### Product version

10.3.2
#### Affected browser(s)
Chrome

**Comment by bm64** [comment]
Hello @zhangmengjie-amy 

I'm not quite sure if I understand how to recreate the issue. Everytime I follow these steps: select point -> select another point -> hover first point, the first point exists. Perhaps you want to select few points at once? If that is the case, you should hold `shift` key and select points.

**Comment by zhangmengjie-amy** [comment]
https://github.com/highcharts/highcharts/assets/113492449/9fbf2816-72da-4d51-b72d-2fd327caa747

I make a screen recording, open the live demo link, and click the point at "march" axis, the point is selected with a black border, and then click the "Forth" axis, the point disappears, and click the point at "May" axis, the point also disappears.

**Comment by zhangmengjie-amy** [comment]
I find this issue only when I set marker enabled to false and set allowPointSelect to true.  For line chart, I hover on a point, the marker display and then click it, it will have a border style, then move mouse to another point, click it, the current clicked point has no border style and its marker disappear. 
If I set marker enabled to true, there is no issue with selected border style. So I close the issue.
